framerail/src/app.d.ts

```diff
@@ -1,9 +1,103 @@
 // See https://kit.svelte.dev/docs/types#app
 // for information about these interfaces
 // and what to do when importing types
+
 declare namespace App {
   // interface Locals {}
-  // interface PageData {}
-  // interface Error {}
+  interface PageData {
+    /** Data about the site. */
+    site: {
+      site_id: string
+      [anySite: any]: unknown
+    }
+    /** Data about the page itself. */
+    page: {
+      page_id: string
+      page_created_at: string
+      page_updated_at?: string
+      page_deleted_at?: string
+      page_revision_count: number
+      site_id: mumber | string
```

Comment:
Typo
```suggestion
      site_id: number | string
```

Also, in what case could this be a string?

Reply:
Not sure why i thought this could be string aswell.